README.md

```diff
@@ -11,7 +11,7 @@ Command line tool to mass upload media folders to your Google Photos account.
 
 # Google Photos uploader CLI   
 
-While the official tool only supports Mac OS and Windows, this brings an uploader to Linux too. Lets you upload photos from, in theory, any OS for which you can compile a Go program.     
+While the official tool only supports Mac OS and Windows, this brings an uploader to Linux too. Lets you upload photos from, in theory, any OS for which you can compile a Go program or build a Docker image using the provided `Dockerfile`.
```

Comment:
Should we add some documentation about the dockered version? 

You have include it on the PR, should we include it on the documentation too?

Reply:
I wasn't sure on how much to document and I settled on minimal.
It depends on if you want to go further with Docker workflow, ie creating an Automated Build, etc. For people used to work with Docker, just having the Dockerfile handy will be all what they'll need, thou.
Anyway, just let me know how much you think is needed. Or let's postpone it after creating an AB. Your choice.

Reply:
@pacoorozco Any thoughts on how should I continue on this?

Reply:
It maybe use [multi-stage dockers](https://tutorialedge.net/golang/go-multi-stage-docker-tutorial/) to reduce the size of the final image.